scripts/makeCheckpoints.ts

```diff
@@ -0,0 +1,141 @@
+import * as grpc from '@grpc/grpc-js'
```

Comment:
Assuming these proto files and makeCheckpoints files are directly copy-pasted from accountbased, then approved.

Reply:
yes, copy pasta'ed